woocommerce/woocommerce
# CYS - Filter out patterns with external dependencies

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/48601



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

⚠️ To test only by devs:
1. Replace these lines https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Blocks/Patterns/PTKPatternsStore.php#L190-L202 by:
```php
'categories' => array(
	'testimonials',
),
```
2. Go to `wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com` and uncheck `Allow usage of WooCommerce to be tracked`.
3. Now recheck it and save.
4. Go to `wp-admin/tools.php?page=action-scheduler`, search for the `fetch_patterns` action, and ensure it was scheduled.
5. Click on `Run` under the action name if it is still pending and wait until it is completed.
6. Go to `wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store`, click `Start designing` or `Customize your theme`.
7. In the assembler, click on `Design your homepage`.
8. Click on the `Reviews` category and check you don't see any broken patterns because of the missing jetpack plugin.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
CYS - Filter out patterns with external dependencies.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
